zigpy/config/__init__.py

```diff
@@ -43,10 +45,13 @@
         ),
         vol.Optional(
             CONF_NWK_EXTENDED_PAN_ID, default=CONF_NWK_EXTENDED_PAN_ID_DEFAULT
-        ): t.ExtendedPanId,
-        vol.Optional(CONF_NWK_KEY, default=CONF_NWK_KEY_DEFAULT): cv_key,
-        vol.Optional(CONF_NWK_PAN_ID, default=CONF_NWK_PAN_ID_DEFAULT): vol.All(
-            cv_hex, t.PanId
+        ): vol.Any(None, t.ExtendedPanId.convert),
+        vol.Optional(CONF_NWK_KEY, default=CONF_NWK_KEY_DEFAULT): vol.Any(None, cv_key),
+        vol.Optional(CONF_NWK_PAN_ID, default=CONF_NWK_PAN_ID_DEFAULT): vol.Any(
+            None, vol.All(cv_hex, t.PanId)
+        ),
+        vol.Optional(CONF_NWK_TC_ADDRESS, default=CONF_NWK_TC_ADDRESS_DEFAULT): vol.Any(
+            None, cv_key
```

Comment:
Should `CONF_NWK_TC_ADDRESS` be a `cv_key`?

Reply:
Doh
copy'n'paste from the key